### ISSUE TITLE
0 Balance and impossible to attach the tangle

### ISSUE THREAD

**Issue by paciang** [iotaledger/wallet#418]
Hi,
I'm trying from 2 hours to generate a new address hoping to find my IOTA Balance.
I'm using the 2.5.3 version of light wallet.
The message appared is still the same: "attaching the tangle"!!

No way to success!... What's happening???

Thanks a lot

**Comment by wonkytonky** [comment]
![image](https://user-images.githubusercontent.com/19390834/32180324-ed2e772a-bd91-11e7-95df-e360ed8ee3cb.png)
try to change  curl method

**Comment by paciang** [comment]
Have I to log out before or not??

**Comment by wonkytonky** [comment]
no, i think not but you can always logout  .. to be sure it takes a few seconds

**Comment by kons233** [comment]
Hi wonkytonky i did what you  showed, when i try to attach to tangle, it says invalid transaction hash

**Comment by kons233** [comment]
update i actually fixed it by using regular CCurl implementation, and switching back to bitfinex 80, every once in a while it still shows invalid transaction hash, mostly it attaches tangles now and i got my full balance back.... maybe you just have to try different hosts, maybe it helps it sync idk, and maybe since i had my initial balance on the old CCurl? thats what it needed? idk but thats how i did it. hope it helps others
